binding/web/src/cheetah.ts

```diff
@@ -89,15 +89,17 @@ export class Cheetah {
 
   private static _cheetahMutex = new Mutex();
 
+  private _isDetached: boolean = false;
+
   private readonly _transcriptCallback: (cheetahTranscript: CheetahTranscript) => void;
-  private readonly _processErrorCallback?: (error: string) => void;
+  private readonly _processErrorCallback?: (error: Error) => void;
```

Comment:
even though this is a good change, I don't think we should change the API until we do our next Cheetah release (which shouldn't be too far away). For now, we may have to live with a string